libraries/AP_Soaring/AP_Soaring.h

```diff
@@ -81,7 +81,8 @@ class SoaringController {
         THERMAL_WEAK,
         ALT_LOST,
         DRIFT_EXCEEDED,
-        GOOD_TO_KEEP_LOITERING
+        GOOD_TO_KEEP_LOITERING,
+        EXIT_COMMANDED
```

Comment:
trailing comma here would be nice.  Simplifies the deltas.

Reply:
Thanks, fixed